airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -129,13 +163,18 @@ def request_params(
         # The log stream use a different params to get data
         # https://developer.okta.com/docs/reference/api/system-log/#datetime-filter
         stream_state = stream_state or {}
-        params = {
-            "limit": self.page_size,
-            **(next_page_token or {}),
-        }
+        params = OktaStream.request_params(self, stream_state, stream_slice, next_page_token)
         latest_entry = stream_state.get(self.cursor_field)
         if latest_entry:
             params["since"] = latest_entry
+            # [TDD] When the cursor value from the stream state is abnormally large, set until
+            # o/w the server side that sets now to until will throw an error:
```

Comment:
what o/w means @YiyangLi can you remove the acronym?

Reply:
Can you give more context about this implementation too?

Reply:
@marcosmarxm this is to address the acceptance test with abnormally large value in state, which is also called [teststatewithabnormallylargevalues](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference#teststatewithabnormallylargevalues)

Under the file `integration_tests/abnormal_state.json`, the state value looks like this 

```
"logs": { "published": "3021-09-08T07:04:28.000Z" }
```

If `until` is not set on the client-side, okta's server-side will use Datetime.now. The parameter validation will throw an error as `since` is abnormally large -- larger than `until` (which is now). The error message is: `The "until" date must be later than the "since" date`

It's just to address the test, the solution is to set the same abnormally large value on until if since is larger than now. It's a good test that handles an abnormal case. I think it's reasonable to run the test on users, groups or groupMembers stream. Running it on logs is a little overkilled. However, if I remove the logs stream out of `incremental` section, all other incremental tests (e.g. [testreadsequentialslices](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference/#testreadsequentialslices)) are not run. I wish there is a way to just skip (teststatewithabnormallylargevalues)[https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference#teststatewithabnormallylargevalues]. Since I can't skip it, I choose to fix it.